sdcm/utils/version_utils.py

```diff
@@ -55,7 +55,7 @@
         (?P<minor>0|[1-9]\d*)
         \.
         (?P<patch>0|[1-9]\d*)
-        (?:-(?P<prerelease>
+        (?:[-.](?P<prerelease>
```

Comment:
This will fix the problem from the SCT side.
But it is not correct thing to do.
This is not correct because it won't be the `SEMVER_REGEX` anymore.